code/datums/wounds/burns.dm

```diff
@@ -164,18 +164,34 @@
 	. = ..()
 	. += "<div class='ml-3'>"
 
+	. += "Wound Bed Damage: [span_deadsay("[flesh_damage]")]\n"
+	if(flesh_healing > 0)
+		. += "Positive signs of healing in the flesh damage, rate: [span_green("[round(flesh_healing, 0.01)]")]\n"
```

Comment:
move this bit into the else on 173, so it won't show up when the wound is fully healed